### DISCUSSION TITLE
FleetDM 'Organization avatar URL' - Company Logo

### DISCUSSION THREAD

**Discussion by adamphetamine** [Security-Onion-Solutions/securityonion#8863] [Unsupported Versions] [answered]
Hi All,
In FleetDM settings there is a spot to add a URL for your company logo.
Fleet calls it 'Organization avatar URL'

This adds a company logo to the top left hand corner of FleetDM interface.

I have it working on a FleetDM VM, but the exact same URL does not import or use the logo in SO.
I have checked and the URL/ logo is publicly accessible, so I'm not sure what to do next- 

- how to solve this please?

**Comment by defensivedepth** [comment, marked as answer]
What version of Security Onion are you using? Is it an air gap install?

I was able to get this working just fine:

![image](https://user-images.githubusercontent.com/954732/195101183-cbefd7bc-0a54-4571-bd8a-59db536508ab.png)

**Comment by adamphetamine** [comment]
Thanks for the confirmation, very odd but I accept that's the answer. Cheers

**Reply by defensivedepth** [reply]
What version of Security Onion are you using? Is it an Air Gap install?

**Reply by adamphetamine** [reply]
oops sorry for not answering- this is not an air gap install. 
It is hosted in the same hosting co that has an earlier (standard) FleetDM installed. I upgraded to Security Onion because of all the extra included features and for some unknown reason this feature (and one other I opened a discussion about) don't seem to work.

I will report back if I find an answer, thanks again.

**Comment by adamphetamine** [comment]
The answer to this turned out to be dumb-
Moving the logo asset to another Server and using TLS fixed it.

ie. Moving from HTTP:// to HTTPS://
